frontend/packages/console-shared/src/components/formik-fields/YAMLEditorField.scss

```diff
@@ -12,5 +12,8 @@
     display: flex;
     flex: 1 0 0;
     margin-left: 20px;
+    @media (max-width: 991px) {
```

Comment:
Would be good to use PF variables: `$pf-global--breakpoint--lg`
I believe their breakpoints are setup for mobile first and therefore using `min-width`.

Reply:
updated it.